.github/workflows/pr-checks.yml

```
@@ -894,3 +894,44 @@ jobs:
     - name: Build code
       run: ./build.sh
     - uses: ./../action/analyze
+
+  test-packaging-javascript-source-root:
```

Minor, but packaging is a different feature so let's try and keep the tests distinct.
```suggestion
  test-javascript-source-root:
```